README.md

```diff
@@ -26,6 +26,90 @@ If the above doesn't help, please [submit an issue](https://github.com/fastlane/
 
 **Note**: If you want to report a regression in _fastlane_ (something that has worked before, but broke with a new release), please mark your issue title as such using `[Regression] Your title here`. This enables us to quickly detect and fix regressions.
 
+## _fastlane_ team
+
+<table>
+<tr>
+<td>
+<img src='https://github.com/lacostej.png?size=200' width=140>
+<h4 align='center'><a href='https://twitter.com/'>Jerome Lacoste</a></h4>
+</td>
+<td>
+<img src='https://github.com/taquitos.png?size=200' width=140>
+<h4 align='center'><a href='https://twitter.com/taquitos'>Joshua Liebowitz</a></h4>
+</td>
+<td>
+<img src='https://github.com/jdee.png?size=200' width=140>
+<h4 align='center'><a href='https://twitter.com/'>Jimmy Dee</a></h4>
+</td>
+<td>
+<img src='https://github.com/DanToml.png?size=200' width=140>
+<h4 align='center'><a href='https://twitter.com/DanToml'>Danielle Tomlinson</a></h4>
+</td>
+<td>
+<img src='https://github.com/mgrebenets.png?size=200' width=140>
+<h4 align='center'><a href='https://twitter.com/mgrebenets'>Maksym Grebenets</a></h4>
+</td>
+</tr>
+<tr>
+<td>
+<img src='https://github.com/matthewellis.png?size=200' width=140>
+<h4 align='center'><a href='https://twitter.com/'>Matthew Ellis</a></h4>
+</td>
+<td>
+<img src='https://github.com/joshdholtz.png?size=200' width=140>
+<h4 align='center'><a href='https://twitter.com/joshdholtz'>Josh Holtz</a></h4>
+</td>
+<td>
+<img src='https://github.com/lmirosevic.png?size=200' width=140>
+<h4 align='center'><a href='https://twitter.com/lmirosevic'>Luka Mirosevic</a></h4>
+</td>
+<td>
+<img src='https://github.com/getaaron.png?size=200' width=140>
+<h4 align='center'><a href='https://twitter.com/getaaron'>Aaron Brager</a></h4>
+</td>
+<td>
+<img src='https://github.com/revolter.png?size=200' width=140>
+<h4 align='center'><a href='https://twitter.com/Revolt666'>Iulian Onofrei</a></h4>
+</td>
+</tr>
+<tr>
+<td>
+<img src='https://github.com/janpio.png?size=200' width=140>
+<h4 align='center'><a href='https://twitter.com/'>Jan Piotrowski</a></h4>
```

Comment:
README is generated? Maybe do once more so will release with current data (except if this is done on release anyway)

Reply:
Yep, auto-generated, part of this PR